controllers/ebpf/agent_controller.go

```diff
@@ -449,38 +449,6 @@ func (c *AgentController) configureFlowFilter(filter *flowslatest.EBPFFlowFilter
 			Value: filter.Protocol,
 		})
 		switch filter.Protocol {
-		case "TCP", "UDP", "SCTP":
-			if filter.SourcePorts.Type == intstr.String {
-				config = append(config, corev1.EnvVar{Name: envFilterSourcePortRange,
-					Value: filter.SourcePorts.String(),
-				})
-			}
-			if filter.SourcePorts.Type == intstr.Int {
-				config = append(config, corev1.EnvVar{Name: envFilterSourcePort,
-					Value: strconv.Itoa(filter.SourcePorts.IntValue()),
-				})
-			}
-			if filter.DestPorts.Type == intstr.String {
-				config = append(config, corev1.EnvVar{Name: envFilterDestPortRange,
-					Value: filter.DestPorts.String(),
-				})
-			}
-			if filter.DestPorts.Type == intstr.Int {
-				config = append(config, corev1.EnvVar{Name: envFilterDestPort,
-					Value: strconv.Itoa(filter.DestPorts.IntValue()),
-				})
-			}
-			if filter.Ports.Type == intstr.String {
-				config = append(config, corev1.EnvVar{Name: envFilterPortRange,
-					Value: filter.Ports.String(),
-				})
-			}
-			if filter.Ports.Type == intstr.Int {
-				config = append(config, corev1.EnvVar{Name: envFilterPort,
-					Value: strconv.Itoa(filter.Ports.IntValue()),
-				})
-			}
-
 		case "ICMP", "ICMPv6":
```

Comment:
lets move icmp filters out of this switch statement and remove the switch statement

Reply:
sure https://github.com/netobserv/network-observability-operator/pull/738/commits/4422d58e4f1ab046018a267ec94666b2897d2715